extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts

```diff
@@ -696,6 +699,16 @@ export class MigrationCutoverDialog {
 			this._lastAppliedBackupInfoField.text.value = this._model.migrationStatus.properties.migrationStatusDetails?.lastRestoredFilename ?? '-';
 			this._lastAppliedBackupTakenOnInfoField.text.value = lastAppliedBackupFileTakenOn! ? convertIsoTimeToLocalTime(lastAppliedBackupFileTakenOn).toLocaleString() : '-';
 
+			if (isBlobMigration) {
+				if (!this._model.migrationStatus.properties.migrationStatusDetails?.currentRestoringFilename) {
+					this._currentRestoringFileInfoField.text.value = '-';
+				} else if (this._model.migrationStatus.properties.migrationStatusDetails?.lastRestoredFilename === this._model.migrationStatus.properties.migrationStatusDetails?.currentRestoringFilename) {
+					this._currentRestoringFileInfoField.text.value = loc.ALL_BACKUPS_RESTORED;
```

Comment:
if currentRewstoringFilename and lastRestoredFilename are both null or blank should this display 'pending' or something different.